test/runtests.jl

```diff
@@ -14,11 +14,7 @@ ENV["GKSwstype"] = "100"
 islinux = Sys.islinux()
 istravis = "TRAVIS" ∈ keys(ENV)
 datadir = joinpath(@__DIR__,"data")
-visualtests = !istravis || (istravis && islinux)
-if !istravis
-  Pkg.add("Gtk")
-  using Gtk
-end
+visualtests = false
```

Comment:
Please undo this change in the final iteration so that we can merge :)

Reply:
Woops, snuck in. Removed.